packages/patterns-react/package.json

```diff
@@ -50,6 +50,7 @@
     "@carbon/ibmdotcom-react": "^1.1.1",
     "@carbon/ibmdotcom-styles": "1.1.1",
     "@carbon/ibmdotcom-utilities": "1.1.1",
+    "@carbon/pictograms-react": "^10.7.0",
```

Comment:
@RaphaelAmadeu we can put this under `devDependencies` now and not `dependencies`